src/FlightMap/MapItems/VehicleMapItem.qml

```diff
@@ -107,6 +112,33 @@ MapQuickItem {
                 origin.y:       vehicleIcon.height / 2
                 angle:          isNaN(heading) ? 0 : heading
             }
+
+            QGCMenu {
+                id: adsbVehicleClickMenu
+
+                QGCMenuItem {
+                    text: qsTr("Hide Vehicle")
+                    onTriggered: {
+                        if (adsbVehicleManager && icaoAddress) {
+                            adsbVehicleManager.setHiddenForADSBVehicle(Number(icaoAddress), true)
+                        } else {
+                            mainWindow.showMessageDialog(qsTr("Error"), qsTr("ADSB vehicle manager or icao address not found"))
+                            console.log("ADSB vehicle manager or icao address not found")
+                        }
+                    }
+                }
+            }
+
+            MouseArea {
+                anchors.fill: parent
+                enabled:      _adsbVehicle
+                cursorShape:  _adsbVehicle ?  Qt.PointingHandCursor : Qt.ArrowCursor
```

Comment:
I agree that the cursor is ugly, but I think it is good UX to change it regardless. Changing the cursor tells the user the vehicle can be clicked, and it could be a nice reminder to see that the cursor changes, in case they forgot you can hide adsb vehicles. But you are right in that cursor change is not really used in QGC, so I am open to removing it if you think we should?

Reply:
If it was consistently used, it could be ok to change it (regardless of it not being very pretty), but I think we should not add special cursor behaviour for this alone.